### PR TITLE
Update MAM benchmark on website

### PR DESCRIPTION
**Pull request by mlober**
This PR corrects the benchmark results for the multi-area model with NEST 3.8 on the NEST website. It separates the strong scaling plot into ground and metastable state.

### CODE REVIEW

**Comment by jessica-mitchell** [comment]
Link to Read the docs build: https://nest-simulator--3358.org.readthedocs.build/en/3358/benchmark_results.html

File: doc/htmldoc/benchmark_results.rst [hunk 1]
```diff
@@ -44,7 +44,11 @@ Strong scaling experiment of the Multi-area-model [5]_
        :class: sd-align-major-center
        :columns: 10
 
-       .. image:: /static/img/mam_benchmark.png
+       Dynamical regime: Ground state
+       .. image:: /static/img/mam_ground-state_benchmark.png
+
+       Dynamical regime: Metastable state
```
Comment (jessica-mitchell) [inline doc/htmldoc/benchmark_results.rst:50]:
```suggestion
       Dynamical regime: Metastable state
       
```

File: doc/htmldoc/benchmark_results.rst [hunk 2]
```diff
@@ -44,7 +44,11 @@ Strong scaling experiment of the Multi-area-model [5]_
        :class: sd-align-major-center
        :columns: 10
 
-       .. image:: /static/img/mam_benchmark.png
+       Dynamical regime: Ground state
```
Comment (jessica-mitchell) [inline doc/htmldoc/benchmark_results.rst:47]:
```suggestion
       Dynamical regime: Ground state
       
```

**Review by jessica-mitchell — CHANGES_REQUESTED** [review]
Images are not rendering- I think it's the lack of new line, added it as suggestion

**Review by jessica-mitchell — APPROVED** [review]

**Review by ackurth — APPROVED** [review]
lgtm

**Pull request merged**